src/components/Calls/CallIFrame.js

```diff
@@ -1,14 +1,18 @@
 import { h } from 'preact';
 
-import { Screen } from '../Screen';
+import { Livechat } from '../../api';
 import { createClassName } from '../helpers';
 import styles from './styles.scss';
 
+const { localStorage } = window;
+const storedState = JSON.parse(localStorage.getItem('store'));
 
-export const CallIFrame = (url) => (
-	<Screen.Content nopadding>
+
+export const CallIframe = () => {
+	const url = `${ Livechat.client.host }/meet/${ storedState.room._id }?token=${ storedState.token }&id=${ storedState.user._id }`;
```

Comment:
JFYI: There will be some changes required here after based on [this](https://github.com/RocketChat/Rocket.Chat/pull/22847#discussion_r683456279) comment. Please coordinate with Dhruv on this one.